Makefile

```
@@ -56,12 +56,24 @@ install-standalone: check-bbr
 
 ## install: Build and install the multiplexer version of celestia-appd into the $GOPATH/bin directory.
 # TODO: Improve logic here and in goreleaser to make it future proof and less expensive.
+define EMBED_BIN
+	if test -f internal/embedding/$$out; then \
+	  echo "Skipping download, existing: $$out"; \
+	else \
+	  wget https://github.com/celestiaorg/celestia-app/releases/download/$(CELESTIA_V3_VERSION)/$$url -O internal/embedding/$$out; \
+	fi
+endef
+
 install: check-bbr
 	@echo "--> Download embedded binaries for v3"
```

[uber nit] even though it's not changed in this PR, I think this should say "Downloading"

```suggestion
	@echo "--> Downloading embedded binaries for v3"
```